staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -1650,7 +1650,12 @@ func (d *PersistentVolumeClaimDescriber) Describe(namespace, name string, descri
 		return "", err
 	}
 
-	events, _ := searchEvents(d.CoreV1(), pvc, describerSettings.ChunkSize)
+	var events *corev1.EventList
+	if describerSettings.ShowEvents {
+		events, _ = searchEvents(d.CoreV1(), pvc, describerSettings.ChunkSize)
+	} else {
```

Comment:
Else is not necessary, please remove it. Can you also add test case?

Reply:
It's done. You can check it now.